notebooks/en/enterprise_dedicated_endpoints.ipynb

```diff
@@ -0,0 +1,924 @@
+{
```

Comment:
<p>"...credit card to the billing settings or your HF organization."</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/huggingface/cookbook/pull/122/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='1'/>